maps/test_map.map

```diff
@@ -38,14 +72,19 @@
     },
     {
       "type": "sprite",
-      "asset": "player",
-      "position": [
-        50,
-        100
+      "asset": "wall3.2",
+      "stretch": "yes",
```

Comment:
Is it possible or bad idea to use boolean values instead of strings?

Reply:
It's not bad idea, but I'm lazy and I don't want to change it